website/blog/2025-04-03-ferretdb-v210-release-performance-observability-improvements.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: ferretdb-v210-release-performance-observability-improvements
+title: 'FerretDB releases v2.1.0 with performance and observability improvements'
+authors: [peter, aleksi]
+description: >
+  We are happy to announce the release of FerretDB v2.1.0, which brings significant performance improvements
+  and observability enhancements.
+image: /img/blog/ferretdb-v2-ga.png
+tags: [release]
+---
+
+![FerretDB releases v2.1.0 with performance and observability improvements](/img/blog/ferretdb-v2-ga.png)
+
+We just released FerretDB v2.1.0, which comes on the back of the successful v2.0.0 GA release.
+
+<!--truncate-->
+
+The latest release brings significant improvements in performance, observability, and addresses key issues to enhance your experience with FerretDB.
+This version is a continuation of our commitment to providing a high-performance, fully open-source alternative to MongoDB, built on the robust foundation of PostgreSQL with DocumentDB extension.
+
+Other important changes changes include a fix for the embeddable package and a resolution to an indexing issue that affected the previous version.
+Let's dive into the details of what you can expect from FerretDB v2.1.0.
+
+## Important upgrade information
```

Comment:
Update, not upgrade. PostgreSQL calls it update; we call it update. Why did we suddenly decide to call a minor version update "upgrade"?